### PR TITLE
add optimus 4kpbmc snRNA test

### PR DESCRIPTION
**Pull request by barkasn**
### Purpose

- Add (manually triggered) test that runs the 4kpbmc dataset as a snRNA-seq dataset

### CODE REVIEW

File: test/optimus_snrna/4kpbmc/test_inputs.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+  "TestOptimusPR.expected_bam_hash": "none",
+  "TestOptimusPR.reference_matrix": "gs://hca-dcp-sc-pipelines-test-data/optimus-test-data/referenceMatrix.rds",
+  "TestOptimusPR.expected_cell_metric_hash": "none",
```
Comment (jessicaway) [inline test/optimus_snrna/4kpbmc/test_inputs.json:4]: No expected hash?

Comment (barkasn) [reply test/optimus_snrna/4kpbmc/test_inputs.json:4]: nope! This is just for running the pipeline with this data the output always gets checked manually. reference matrix would be empty too if it didn't break cromwell -- maybe someone knows a way to fix this?

Comment (barkasn) [reply test/optimus_snrna/4kpbmc/test_inputs.json:4]: Maybe I should actually adjust this so it doesn't run the checker at all -- but actually we do the same thing for the 4kpbmc manual test so maybe that is another ticket.

Comment (jessicaway) [reply test/optimus_snrna/4kpbmc/test_inputs.json:4]: Maybe we could rename it to indicate that this is a manual test?

**Pull request merged**